test/runner/lib/executor.py

```diff
@@ -1115,7 +1115,7 @@ def integration_environment(args, target, cmd, test_dir):
 
     integration = dict(
         JUNIT_OUTPUT_DIR=os.path.abspath('test/results/junit'),
-        ANSIBLE_CALLBACK_WHITELIST='junit',
+        ANSIBLE_CALLBACK_WHITELIST='junit, aws_resource_actions',
```

Comment:
This should only be enabled for AWS tests. Append it to the environment variable here instead:

https://github.com/ansible/ansible/blob/332dc6a93a1d791270fba012c8ebb28f3a70a81f/test/runner/lib/cloud/aws.py#L87